server/blob/blob-memory/src/main/java/org/apache/james/blob/memory/MemoryBlobStore.java

```diff
@@ -40,12 +40,12 @@
 
 public class MemoryBlobStore implements BlobStore {
     private final BlobId.Factory factory;
-    private final List<MemoryBucket> buckets;
+    private final ArrayList<MemoryBucket> buckets;
 
     @Inject
     public MemoryBlobStore(BlobId.Factory factory) {
         this.factory = factory;
-        buckets = Arrays.asList(new MemoryBucket(BucketName.DEFAULT));
+        buckets = new ArrayList<>(Arrays.asList(new MemoryBucket(BucketName.DEFAULT)));
```

Comment:
I think it should be empty at first. Currently, creating the default bucket is performed inside Guice Module. Or you should have a test about default bucket should be existed after initializing (but I don't like it)

Reply:
No guice bindings yet at that stage, and those are not integration tests... Explain to me how I'm supposed to initialize it otherwise???

Reply:
I realized that maybe I misunderstood and missed your point at first. Took it out of `MemoryBlobStore` and initialized it in the setup of `MemoryBlobStoreTest`. I think that's what you meant at the end?